### ISSUE TITLE
Discord Rich Presence integration not working in Linux

### ISSUE THREAD

**Issue by pa88word** [headsetapp/headset-electron#428]
### Description
When Discord Rich presence integration is enabled in the Headset Music app discord does not recognize the app at all. (Other apps have been tested and rich presence integration is working on those apps.)
A clear and concise description of what the issue is about.

### Expected behaviour
That Discord would detect, at the very least, the app being open if not showing what song is playing in the headset app.

### Actual behaviour
Discord Rich presence integration is enabled but Discord does not detect the app in any way shape or form.

### Your Environment (DE)
Zorin OS 16 desktop (I believe based on Xfce)
- What OS are you using?
Zorin OS 16 (Ubuntu 20.04 based)
- What version of Headset? (copy and paste from the user profile page if possible)
Core: v2.11.0, Wrapper: v3.3.3
### Screenshots

🖼
![image](https://user-images.githubusercontent.com/94565314/142271461-e48932de-4466-4d5a-9001-afc416faf0db.png)

![image](https://user-images.githubusercontent.com/94565314/142271533-fc5bf3b5-3bd9-4752-bb86-e21b02bcd940.png)

**Comment by fcastilloec** [comment]
Hi, sorry for not getting back to you sooner, but I've been on holiday. I'll take a look and see if I can reproduce the error.
From the screenshots, I can't see if you're playing any songs. Discord only shows something when a song is playing, otherwise, there's no need to show anything.

**Comment by pa88word** [comment]
No problem man. I have confirmed that it works perfectly on windows except for when titles are too long then it just says the name of the last song and the time played of the song at 0:00. 
I have also discovered that (as with spotify) it does not show up any longer under the "Activity status" pagep under settings but instead is a status that can be seen when clicking on one's profile. 

Unfortunately it still is not working on my distribution of Linux.

**Comment by fcastilloec** [comment]
I have tested the app on ZorinOS and I can confirm that it's working as intended. I haven't been able to find a song with a long title so I haven't been able to test that. If you have the link to the song you mentioned, I can take a look at that.

As you mentioned, it was never intended to be shown under "Activity status", only under the user's profile. Here's a screenshot of Headset and Discord working on Zorin OS 16.

![Selection_002](https://user-images.githubusercontent.com/1280530/144149809-bdc5fffe-9852-46ea-b5e3-7ab8eacea12a.png)

**Comment by pa88word** [comment]
Hmm. Must be an issue that I am distinctly having. There is a possibility that better discord somehow affects it as well. 

As far as the title thing, that was on Windows on a non-modded version of discord. Music that has a long title like this: https://www.youtube.com/watch?v=dXItrIerZdU do not show up, as previously mentioned.

**Comment by fcastilloec** [comment]
I tried the song with the long title, and I found a couple of other ones that were long. They all work... I tested on Ubuntu and Zorin OS. Quick question, how did you install Discord? Maybe there's a problem with snap or flatpak. All my installations are from the official supported `.deb` package provided directly from Discord. I suggest you use the same.

![Selection_002](https://user-images.githubusercontent.com/1280530/144299200-6a4008e1-df1b-46d3-9ee2-42f0c3987dee.png)
